app/views/records/edit_fields/attribution_partials/_select_field.html.erb

```diff
@@ -0,0 +1,14 @@
+<div class="form-group">
+	<% options = { class: "control-label multi_value optional" } %>
+	<%= label_tag "#{template}_#{field_slug}", options do %>
```

Comment:
I think Chris mentioned we can use simple_form to build the forms. It would help us get `label` tags automatically, placeholder items, popovers with contextual_info, etc with i18n support. We would also get rid of a lot of code that simple_form would do for us, right?